profiles/forms/quota_forms.py

```diff
@@ -21,12 +21,17 @@ def __init__(self, *args, **kwargs):
             for attribute_definition in all_attributes:
                 # get the value if exist already
                 current_quota = Quota.objects.filter(scope=self.scope, attribute_definition=attribute_definition).first()
-                default_value = current_quota.limit if current_quota is not None else None
+                default_value=None
+                consumed = None
+                if current_quota is not None:
+                    default_value = current_quota.limit
+                    consumed = current_quota.consumed
                 self.fields[f"attribute_definition_{attribute_definition.id}"] = \
                     IntegerField(label=attribute_definition.name,
                                  min_value=0,
                                  required=False,
                                  initial=default_value,
+                                 help_text=f"Already consumed at team level: {consumed}",
```

Comment:
f"Already consumed at team level: {consumed}" if consumed is not None else consumed